helpers/for.js

```diff
@@ -3,8 +3,8 @@
 const _ = require('lodash');
 
 const factory = () => {
-    return function(from, to, context) {
-        const options = arguments[arguments.length - 1];
+    return function(from, to, context, options) {
+        options =  options ? options : (context ? context : to);
```

Comment:
It's not immediately apparent how this relates to the parameter shuffling that takes place below and why we wouldn't do this at the same time.

Reply:
added a comment